war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2226,54 +2198,6 @@ function buildFormTree(form) {
     }
 }
 
-var hoverNotification = (function() {
```

Comment:
Should be migrated as a separate change. Especially that there sometimes are multiple tooltips shown at the same time because of this is weird.

Reply:
From my (limited) testing it seems to be working alright, are there any situations you've found where it doesn't?

https://user-images.githubusercontent.com/43062514/187247513-efb5ba60-7333-4cd8-863c-b2d1fb3ec375.mov